### PR TITLE
Solved issue #1361

### PR DESCRIPTION
**Pull request by mshubhankar**
Changes made:
-> Ported the find-procedure popup to iziModal style.
-> Fixed the cancel button.
-> Save button name changed to Refresh.

### CODE REVIEW

File: interface/orders/orders_results.php [hunk 1]
```diff
@@ -208,17 +208,41 @@ function QuotedOrNull($fld) {
...
+ // dlgopen('types.php?popup=1&order=' + f[ptvarname].value, '_blank', 800, 500); // This calls the old style popup
+ event.preventDefault();
+    console.log(event.target.href);
+    $("#modal-iframe").iziModal({
+    title: 'Find procedure',
+    subtitle: 'Select procedure from the table below', headerColor: '#88A0B9',
+    iframe: true,
+    iframeHeight: 500,
+    width: 700,
+    height: 500,
```
Comment (muarachmann) [inline interface/orders/orders_results.php]: @mshubhankar good work done here. Just some few stuffs to be done. izi headers on the other modals

Comment (mshubhankar) [reply interface/orders/orders_results.php]: I'm sorry I couldn't fully understand. Which other modal are you exactly taking about ? Is it the Add button or Add top level modal ?

**Comment by muarachmann** [comment]
The add

**Comment by mshubhankar** [comment]
The add and add top level iziModals are written in the same definition. I have added a generic header which will work for both modals.

File: interface/orders/types.php [hunk 2]
```diff
@@ -191,6 +191,8 @@ function refreshFamily(id, haskids) {
     event.preventDefault();
     console.log(event.target.href);
     $("#modal-iframe").iziModal({
+    title: 'Add procedure',
+    subtitle: 'Fill in procedure', headerColor: '#88A0B9',
```
Comment (muarachmann) [inline interface/orders/types.php]: a break here will do .... headerColor is a key value

Comment (mshubhankar) [reply interface/orders/types.php]: Yes you are right. I have made the necessary changes to both new definitions.

**Comment by muarachmann** [comment]
sorry @mshubhankar havent had time, testing this in a bit

**Comment by mshubhankar** [comment]
@muarachmann Haven't heard from you in a while. I just wanted to check if everythings okay with the changes I made.

**Comment by muarachmann** [comment]
> @muarachmann Haven't heard from you in a while. I just wanted to check if everythings okay with the changes I made.

Yes actually have been pretty busy!. I think your PR is cool will review it some other time please. You can solve other issues :) Thanks and good work

File: interface/orders/types.php [hunk 3]
```diff
@@ -192,6 +191,9 @@ function refreshFamily(id, haskids) {
     event.preventDefault();
     console.log(event.target.href);
     $("#modal-iframe").iziModal({
+    title: 'Add procedure',
+    subtitle: 'Fill in procedure',
```
Comment (aethelwulffe) [inline interface/orders/types.php]: Translations?  Meaning use of xlt() function.

Comment (muarachmann) [reply interface/orders/types.php]: yes use those i forgot thanks @aethelwulffe

**Comment by aethelwulffe** [comment]
Also, the way you called the libraries is pretty much the common way it is done, but I believe something like this:
	<?php call_required_libraries(['bootstrap', 'jquery-min-1-9-1']); ?>
Also works fine without using array()

File: interface/orders/orders_results.php [hunk 4]
```diff
@@ -208,17 +208,42 @@ function QuotedOrNull($fld) {
...
+<?php call_required_libraries(array("jquery-min-3-1-1","bootstrap", "iziModalToast")); ?>
 <script language="JavaScript">
 
 
-// This invokes the find-procedure-type popup.
+// This invokes the find-procedure-type popup. Old popup style iframe replaced with iziModal. 
 var ptvarname;
+// function sel_proc_type(varname) {
+//  var f = document.forms[0];
+//  if (typeof varname == 'undefined') varname = 'form_proc_type';
```
[the remark this answers is not in the document]
Comment (mshubhankar) [reply interface/orders/orders_results.php]: Yeah, you're right. I will remove the junk.

File: interface/orders/types.php (hunk 3 above)
Comment (mshubhankar) [reply interface/orders/types.php]: Sorry but I do not understand fully.

Comment (muarachmann) [reply interface/orders/types.php]: xlt() is a translation function @mshubhankar so basically just pass the texts as parameter to it so it displays a correct message in the appropriate language

**Comment by mshubhankar** [comment]
@aethelwulffe Yeah the libraries can be called without array() as well. I refered to some previous code and make it this way. Here are some of the instances :
screenshot 2019-03-08 at 1 30 17 pm

**Comment by aethelwulffe** [comment]
Yeah, you did good, it was just that when I wrote the function I assumed that it would be passed as an array without anyone needing to be explicit.  Since then most all the implementations have been explicit.  Just now noticed.

**Comment by aethelwulffe** [comment]
Mua,
Review, merge.

**Comment by mshubhankar** [comment]
Hey @muarachmann, I hadn't pushed my last commit. I had to make some changes in the PR suggested by @aethelwulffe.

**Comment by mshubhankar** [comment]
I have created another PR with the uncommitted files. Kindly refer #1424

**Pull request merged**